website/pages/blog/guide-aws-org-delegation.mdx

```diff
@@ -0,0 +1,376 @@
+* SCPs do not apply to the management account.
+* The management account has Organization visibility into member accounts and other reconnaissance activities.
+* The management account has Organization management features such as SCPs, enabling/disabling security features across the Organization and all member accounts.
+
+Thus, an attacker may find the Organization Management Account a preferred target to gain a foothold in an enterprise's AWS environments.  From the above, if an attacker is able to gain access to an IAM principal with `sts:AssumeRole` privileges in the organization management account, they could have administrative access to every member account in the entire AWS Organization.  Additionally, the Organization Management account can be subject to misconfiguration that has a greater blast radius and can affect multiple member accounts and applications.  An example of this could be a misconfigured Service Control Policy (SCP).
+
+By delegating management to member AWS Accounts, this reduces the blast radius and follows [AWS Prescriptive Guidance regarding the management account](https://docs.aws.amazon.com/prescriptive-guidance/latest/security-reference-architecture/management-account.html).  AWS recommends only using the management account for tasks that can only be performed by that account.  By delegating management, permissions and access to the management account can be reduced to a minimum.  AWS recommends using both trusted access and delegated administrator to reduce permissions and limit access to the management account.
+
+### What Delegated Administrator Cannot Do
+There are activities that delegated administrators cannot do and thus will still require usage of the organization management account.
+
+The list below is not exhaustive and gives examples of activities that still require usage of the organization management account.
+
+* Moving accounts between OUs.
+* Inviting an account to join the organization.
+* Creating new accounts within the organization.
+* Creating new organizational units within the organization.
+
+### Risks of Delegated Administrator Accounts
+While Delegated Administrator Accounts do reduce risk of overuse of the organization management account, risks of delegated administrator accounts include the following:
+
+* Overprivileged delegated administrator access.  Delegated Administrator accounts could have more relaxed access control.  This could be true if the delegated administrator accounts also host application workloads.
+* Delegated Administrator accounts are impacted by Service Control Policies (SCPs).  There is a risk of potential adverse impact if SCPs are misconfigured.
+
+## Delegated Administrator Management
+
+### Setup: How does Delegation Happen?
+
+There are 3 different ways to delegate organization management.  In this case, all 3 may delegate different abilities and are not mutually exclusive.
```

Comment:
This might be a completely different blog post, but it would be good to talk about why someone should use each method and the pros/cons of each

Reply:
Yea - that gets into use cases for delegated administrators which could be a great follow up post.

Reply:
Yea - that gets into use cases for delegated administrators which could be a great follow up post.